bundles/EcommerceFrameworkBundle/PricingManager/RuleInterface.php

```diff
@@ -79,9 +79,9 @@ public function getDescription($locale = null);
     public function setCondition(ConditionInterface $condition);
 
     /**
-     * @return ConditionInterface
+     * @return ConditionInterface|null
      */
-    public function getCondition();
+    public function getCondition(): ?ConditionInterface;
```

Comment:
missing note in upgrade notes

Reply:
add it here: https://github.com/pimcore/pimcore/pull/9067/commits/054e8f6cbd1b3dca2c1103c0e7e6a13f3bd0a53b